pkg/controlplane/controller/leaderelection/leaderelection_controller_test.go

```diff
@@ -174,7 +175,7 @@ func TestReconcileElectionStep(t *testing.T) {
 						LeaseName:           "component-A",
 						EmulationVersion:    "1.20.0",
 						BinaryVersion:       "1.20.0",
-						PingTime:            ptr.To(metav1.NewMicroTime(fakeClock.Now().Add(-1 * time.Millisecond))),
+						PingTime:            ptr.To(metav1.NewMicroTime(fakeClock.Now())),
```

Comment:
For the reviewer: this is the one not needed anymore because we now check for >=, no >